src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutesSpec.scala

```diff
@@ -38,8 +38,6 @@ class LeoRoutesSpec extends FlatSpec with ScalatestRouteTest with TestLeoRoutes
   it should "200 if you're on the whitelist" in isolatedDbTest {
     Get(s"/api/isWhitelisted") ~> timedLeoRoutes.route ~> check {
       status shouldEqual StatusCodes.OK
-
-      validateCookie { header[`Set-Cookie`] }
```

Comment:
Why do we no longer need this?

Reply:
Good catch. I removed this when I was using the api-endpoint approach, but it should probably be added back.